src/main/java/com/balloon_spring_jpa/balloon/exception/StockBalanceException.java

```diff
@@ -0,0 +1,8 @@
+package com.balloon_spring_jpa.balloon.exception;
+
+public class StockBalanceException extends RuntimeException{
+
+    public StockBalanceException(int quantity) {
+        super("stock balance of this kind of balloons less then in your order on: " + quantity);
```

Comment:
"Stock balance of balloon by id: " + balloonId + " less than ordered quantity: " + quantity

in this case, you need to add UUID baloonID in the method parameters.